hack/cmd_util.sh

```diff
@@ -147,41 +147,50 @@ os_cmd_internal_tmperr="${os_cmd_internal_tmpdir}/tmp_stderr.log"
 # any error exiting settings or traps set by upstream callers by masking the return code of the command 
 # with the return code of setting the result variable on failure.
 function os::cmd::internal::expect_exit_code_run_grep() {
-	cmd=$1
+	local cmd=$1
 	# default expected cmd code to 0 for success
-	cmd_eval_func=${2:-os::cmd::internal::success_func}
+	local cmd_eval_func=${2:-os::cmd::internal::success_func}
 	# default to nothing 
-	grep_args=${3:-} 
+	local grep_args=${3:-} 
 	# default expected test code to 0 for success
-	test_eval_func=${4:-os::cmd::internal::success_func}
+	local test_eval_func=${4:-os::cmd::internal::success_func}
 
 	os::cmd::internal::init_tempdir
 
-	echo "Running  ${cmd}..."
-	
-	cmd_result=$( os::cmd::internal::run_collecting_output "${cmd}"; echo $? )
-	cmd_succeeded=$( ${cmd_eval_func} "${cmd_result}"; echo $? )
+	local name=$(os::cmd::internal::describe_call "${cmd}" "${cmd_eval_func}" "${grep_args}" "${test_eval_func}")
+	echo "${name}..."
```

Comment:
The output as it reads now is more expressive and `Running` doesn't add anything:
```
test/cmd/export.sh:14: executing 'oc new-app -f examples/sample-app/application-template-stibuild.json --name=sample' expecting success...
```